### DISCUSSION TITLE
How to delete a repository generated from a template repository

### DISCUSSION THREAD

**Discussion by kawau1** [community/community#125635] [GitHub Education]
### Hi everyone,

I want to delete the repository generated from education/Series-Intro-to-GitHub-Flow. However, the generated repositories do not show a settings tab and cannot be deleted. Please tell me how to delete it.

**Comment by pedrolemos4** [comment]
If you cannot see the Settings tab in the repository means that you do not have permissions for it. Maybe you are not logged in or just don't have permissions

**Reply by kawau1** [reply]
How do I remove the repository in question from Repositories in the left sidebar or make it invisible?
You will not see a tab for settings in this repository.

![スクリーンショット 2024-05-27 150234 copy](https://github.com/community/community/assets/119904016/8131da56-aba1-49e7-ab1f-864119c24977)

**Reply by pedrolemos4** [reply]
Send a print of the whole page when you are in the repository

**Comment by kawau1** [comment]
![image](https://github.com/community/community/assets/119904016/5cdf98ec-a5ba-436c-b30b-d6a16ea4eb69)

**Reply by pedrolemos4** [reply]
You don't haver permissions for that repo. 
I can't even see it in your profile...

**Reply by pedrolemos4** [reply]
You can see it there because those are the recent repositories you have seen

**Reply by kawau1** [reply]
> You don't haver permissions for that repo. I can't even see it in your profile...

Why is it listed in the Repository section?

**Reply by pedrolemos4** [reply]
Because it's on the repositories that you have seen recently

**Reply by kawau1** [reply]
Will it disappear from my profile if I let it sit for a while?

**Comment by namdng09** [comment]
Hi there, i think that i found the solution for that problem.

You don't have permission to delete the repository because you aren't the owner but it still display on your Repositories because you have been added to contributors. 

![image](https://github.com/community/community/assets/144111926/e7a943ee-94b5-4a74-9ad2-124b2691d4e8)
![image](https://github.com/community/community/assets/144111926/04129fe5-e1f8-4e19-8224-da8551746202)

So that, to do not display, all of things what you can do by Removing yourself from a collaborator's repository.

1) In your setting, move to "Code, planning, and automation" section of the sidebar, then click Repositories.
![image](https://github.com/community/community/assets/144111926/91218ed7-33c1-4fe1-8a0a-8cbdcd03bd5c)

2) Next to the repository you want to leave, click Leave.
![image](https://github.com/community/community/assets/144111926/b190874a-141d-45f0-bd8b-dee3b76aba87)

3) Read the warning carefully, then click I understand, leave this repository.

Thankyou for reading!

**Reply by kawau1** [reply]
Hi @namdng09, Thank you for answering my question.
I was able to delete the repository from the repository section of the configuration as you described.
However, my account does not disappear from the deleted repository contributors.
Does this mean that it has not been removed?
